### ISSUE TITLE
vagrant up hangs on Windows 10 + VMWare Workstation provider

### ISSUE THREAD

**Issue by JeffMelton** [hashicorp/vagrant#9150]
### VMWare Workstation
Version: 14.0.0 build-6661328
vagrant-vmware-workstation-5.0.2

### Vagrant version
Vagrant 2.0.1

### Host operating system
Microsoft Windows [Version 10.0.16299.19]

### Guest operating system
CentOS 7

### Vagrantfile
```ruby
# -*- mode: ruby -*-
# vi: set ft=ruby :
#
Vagrant.configure("2") do |config|
  config.vm.box = "centos/7"
end
```

### Debug output
https://gist.github.com/JeffMelton/7c6f4882fb760dd4fde2c1ad91d6a60f

### Expected behavior
Vagrant should repeatably start the box.

### Actual behavior
Vagrant fails to start the box… usually. I can't predict when it _will_ work, but it does… sometimes. >.< I've noted that there is a Scheduled Task named "vagrant-vmware-desktop-\<computer name\>-\<user name\>" that has a "Queued" status, and the debug output suggests Vagrant is waiting on that task for some reason. 

![vagrant-scheduled-task](https://user-images.githubusercontent.com/165736/32518770-3b35fc7e-c3d0-11e7-9a1b-53e1fdaedea6.png)

`vagrant plugin repair`, `vagrant plugin expunge` (and reinstall) are ineffective. Restarting the PC seems sometimes to break it loose, but not reliably. Reinstalling vagrant has also not been effective.

### Steps to reproduce
1. `vagrant box add centos/7`
2. `vagrant init`
3. `vagrant up` (get a bucket of popcorn and wait… forever)

**Comment by TonyApuzzo** [comment]
What ssh client do you have in your path by default?

In my experience, if the path to ssh.exe has a space in it then vagrant will hang forever. 

I am using the git ssh client, but I manually installed git into `c:\tools\git` instead of `c:\program files\git`
```PowerShell
PS C:\Users\tapuzzo> get-command -All ssh.exe

CommandType     Name                                               Version    Source
-----------     ----                                               -------    ------
Application     ssh.exe                                            0.0.0.0    C:\tools\git\usr\bin\ssh.exe
```

Also you will be sad if you have WSL or OpenSSH-Win64 ssh in your path before git.  Vagrant has an embedded ssh.exe file as well but it doesn't seem to be used.  I have not tried to force that one by messing with the PATH. Cygwin supposedly works too.

**Comment by JeffMelton** [comment]
I do have OpenSSH-Win64 installed and WSL enabled. And the former's ssh binary is first in $env:PATH. Note that I'm never getting to `vagrant ssh` -- the box doesn't start at all. The two times (out of a dozen or more) that `vagrant up` _was_ successful, I was able to `vagrant ssh` with no difficulty. There's also very little in the debug output relating to SSH and nothing that screams "can't find ssh!" Is there something I'm missing, or am I right in thinking this is likely not the issue?

**Comment by TonyApuzzo** [comment]
I don't know, I just had total vagrant lock-ups with paths having spaces. Usually network problems will show a timeout rather than just a total hang.  But I see there are other open issues with vmware provider hanging, maybe those are more aligned with your issue: https://github.com/hashicorp/vagrant/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aopen%20hang  (I'm just another user trying to help, I don't have any connection with Vagrant.)

**Comment by JeffMelton** [comment]
Thanks, @TonyApuzzo. :) I appreciate it.

**Comment by JeffMelton** [comment]
All right, I think I've figured out what the problem is:

It appears that the scheduled task is getting created with a few defaults that are problematic on a mobile workstation, particularly "Start the task only if the computer is on AC power".

**Comment by StefanScherer** [comment]
Today I got a new company notebook with Windows 10 Enterprise, Vagrant 2.0.1, VMware Workstation 14 Pro. So I installed my vagrant license and the vagrant plugin 5.0.4 and tried my first  `vagrant up`.

It hangs see https://gist.github.com/seal-ss/562337aaf2bd9d4322dc8c14e564faf2, the notebook is running in battery mode, the scheduled task has no condition for AC power only, the history shows me that the task has been started.

**Comment by chrisroberts** [comment]
Hi there,

A new VMware plugin has been released today. You can read more about it here:

https://www.hashicorp.com/blog/introducing-the-vagrant-vmware-desktop-plugin

If you are still encountering issues after upgrading, please open a new issue and I will investigate further.

Cheers!
